packages/veritone-widgets/src/redux/modules/mediaDetails/saga.js

```diff
@@ -777,11 +797,150 @@ function* watchLoadTdoRequest() {
   });
 }
 
+function* uploadImage(fileToUpload, widgetId) {
```

Comment:
we should be abstracting generic things to redux-common sooner rather than later.